modules/ROOT/pages/config-mutual-auth.adoc

```diff
@@ -0,0 +1,21 @@
+= Mutual Authentication with Anypoint Platform
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+With Anypoint Runtime Fabric version 1.3.60 or later, you can use the Runtime Fabric *Settings* page in Runtime Manager to enable mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane. Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. Runtime Fabric instances created after July 8, 2020 have the mutual authentication setting enabled by default.
```

Comment:
"between Runtime Fabric and Anypoint Platform"
But you wrote "Anypoint control plane" in the previous sentence.
Do you even need to repeat the names? Could you just say "between both"?